causalpy/pymc_experiments.py

```diff
@@ -321,18 +324,353 @@ def plot(self, counterfactual_label="Counterfactual", round_to=None, **kwargs):
 
         return fig, ax
 
-    def summary(self, round_to=None) -> None:
+    def summary(
+        self, round_to=None, version: str = "coefficients", **kwargs
+    ) -> Union[None, pd.DataFrame]:
         """
         Print text output summarising the results
 
         :param round_to:
             Number of decimals used to round results. Defaults to 2. Use "None" to return raw numbers.
         """
 
-        print(f"{self.expt_type:=^80}")
-        print(f"Formula: {self.formula}")
-        # TODO: extra experiment specific outputs here
-        self.print_coefficients(round_to)
+        if version == "coefficients":
+            print(f"{self.expt_type:=^80}")
+            print(f"Formula: {self.formula}")
+            # TODO: extra experiment specific outputs here
+            self.print_coefficients()
+        elif version == "intervention":
```

Comment:
So I'm not 100% sure about this. At the moment we have a `summary` method for all classes which provides an estimate of the model coefficients. The approach used here is to also call the `summary` method but change what it does completely by changing a kwarg. 

I'm thinking that it might be cleaner to completely separate this and just have a new method called `power_analysis_summary`, or `intervention_summary`, whatever.